Gems/Atom/RHI/Vulkan/Code/Source/RHI/DescriptorSetLayout.h

```diff
@@ -76,7 +76,7 @@ namespace AZ
             const AZStd::vector<VkDescriptorBindingFlags>& GetNativeBindingFlags() const;
             const RHI::ShaderResourceGroupLayout* GetShaderResourceGroupLayout() const;
 
-            static const uint32_t MaxUnboundedArrayDescriptors = (1024 * 1024 * 2); // 2M
+            static const uint32_t MaxUnboundedArrayDescriptors = 900000; //Using this number as it needs to be less than maxDescriptorSetSampledImages limit of 1048576
```

Comment:
Is there any way to represent this magic number? perhaps a centralized location where we keep our limits?

Reply:
yeah. This is not the best but I can add a task to data drive this number from setting registry as oppose to being hardcoded.